README.md

```diff
@@ -144,6 +144,13 @@ The shared storage worklet invocation methods (`addModule`, `run`, and `selectUR
         *   `keepAlive` (defaults to false), a boolean denoting whether the worklet should be retained after it completes work for this call.
             *   If `keepAlive` is false or not specified, the worklet will shutdown as soon as the operation finishes and subsequent calls to it will fail.
             *   To keep the worklet alive throughout multiple calls to `run()` and/or `selectURL()`, each of those calls must include `keepAlive: true` in the `options` dictionary.
+        *   `savedQuery` (defaults to the empty string), a string naming the query to be saved or reused.
```

Comment:
The `resolveToConfig` attribute supported in `options` isn't described in this section.

Reply:
Good point, thank you. We will address in a separate PR. Here is an issue to keep track: #177.